cvmfs/swissknife_notify.cc

```diff
@@ -81,7 +81,7 @@ int CommandNotify::Main(const ArgumentList& args) {
   } else {  // subscribe
     std::string topic = *args.find('t')->second;
     bool continuous = args.count('c') > 0;
-    int revision = GetMinRevision(args);
+    uint64_t revision = GetMinRevision(args);
```

Comment:
```suggestion
    const uint64_t revision = GetMinRevision(args);
```
clang tidy complains: 
```
/home/sftnight/jenkins/workspace/CvmfsTidy/label/docker-x86_64/cvmfs/cvmfs/swissknife_notify.cc:84:5: error: variable 'revision' of type 'uint64_t' (aka 'unsigned long') can be declared 'const' [misc-const-correctness,-warnings-as-errors]
    uint64_t revision = GetMinRevision(args);
    ^
             const 
```